node/src/components/deploy_acceptor.rs

```diff
@@ -492,12 +492,15 @@ impl DeployAcceptor {
                         verification_start_timestamp,
                     })
             }
-            ExecutableDeployItemIdentifier::Package(ContractPackageIdentifier::Hash {
-                contract_package_hash,
-                version: maybe_package_version,
-            }) => {
+            ExecutableDeployItemIdentifier::Package(
+                ref contract_package_identifier @ ContractPackageIdentifier::Hash {
+                    contract_package_hash,
+                    ..
+                },
+            ) => {
                 let query_key = Key::from(contract_package_hash);
                 let path = vec![];
+                let maybe_package_version = contract_package_identifier.version();
```

Comment:
Is this change related to removal of `NodeId`?

Reply:
No, this was in the merge conflict. I'm not exactly sure if this is correct, but I recall it seeming like the most rational thing at the time.